<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="postCats">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="postCatName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="postCatActive">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Aktiv</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newPostButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Ny post</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Lagre</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
